text/0032-distinct-attribute.md

```diff
@@ -0,0 +1,238 @@
+- Title: Distinct Attribute
+- Start Date: 2021-04-16
+- Specification PR: [#32](https://github.com/meilisearch/specifications/pull/32)
+- MeiliSearch Tracking-Issues: [milli#168](https://github.com/meilisearch/milli/issues/168)
+
+# Distinct Attribute
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+The distinct attribute is usefull to discard other occurences of document having the same value as the field setted as a distinct attribute.
+
+The value of a field whose attribute is set as a distinct attribute will always be unique in the returned documents.
+
+### II. Motivation
+
+The new search engine called Milli no longer processes the distinct attribute as the current MeilliSearch. This specification makes it possible to state that Milli will be as backwards compatible as possible with the v0.20 in the usage of API interfaces but also in the expected search results.
+
+### III. Additional Materials
+
+#### Algolia
+
+Algolia distinct feature is based on one attribute, as defined in `attributeForDistinct`. It is possible to limit the number of returned records that contains the same attribute value. It is done at indexing time.
```

Comment:
```suggestion
Algolia distinct feature is based on one attribute, as defined in `attributeForDistinct`. Adding an attribute to this setting will make that all search results have a different value in the given attributes field.

This is done at processing time.
```

What is processing time? On search? On indexing ? Sorry ><

Reply:
Indexing. I will precise that! Thank you

Reply:
I've just see that you added processing time when I used indexing time before!